### PR TITLE
Pa suggestions

### PR DESCRIPTION
**Pull request by pchampin**
Those are really just suggestions. You can cherry pick those you like and reject others... But I think that deserved discussion.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/json-ld-streaming/pull/10.html" title="Last updated on Mar 25, 2020, 4:34 PM UTC (2e44e08)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/json-ld-streaming/10/04e55a0...2e44e08.html" title="Last updated on Mar 25, 2020, 4:34 PM UTC (2e44e08)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -218,11 +218,7 @@ <h3 id="streaming-document-examples">Examples</h3>
               <h3 id="streaming-document-examples-valid">Valid Examples</h3>
               <pre class="example" title="Valid streaming document with @context, @id, and other properties">
                 {
-                  "@context": {
-                    "name": "http://schema.org/name",
-                    "url": "http://schema.org/url",
-                    "image": "http://schema.org/image"
-                  },
+                  "@context": "http://schema.org/",
```
Comment (rubensworks) [inline index.html]: I made the context more verbose to keep the examples fully standalone.
But I don't have a strong opinion on this, so I'm fine with changing it.

File: index.html [hunk 2]
```diff
@@ -302,13 +280,7 @@ <h3 id="streaming-document-examples-valid">Valid Examples</h3>
               </pre>
               <pre class="example" title="Valid streaming document with @context, plain @type, @id, and other properties">
                 {
-                  "@context": {
-                    "Person": "http://schema.org/Person",
-                    "name": "http://schema.org/name",
-                    "url": "http://schema.org/url",
-                    "image": "http://schema.org/image"
-                  },
-                  "@id": "https://www.rubensworks.net/#me",
```
Comment (rubensworks) [inline index.html]: I think this @id was removed by mistake.

File: index.html (hunk 1 above)
Comment (pchampin) [reply index.html]: I understand that, but so doing, you also changed their semantics, to something that may be surprising/disturbing to some readers. Hence my proposal

File: index.html (hunk 2 above)
Comment (pchampin) [reply index.html]: Oops, indeed, that was a mistake...

File: index.html [hunk 3]
```diff
@@ -302,13 +280,7 @@ <h3 id="streaming-document-examples-valid">Valid Examples</h3>
...
+                  "@context": "http://schema.org/",
```
Comment (pchampin) [inline index.html]:
```suggestion
                  "@context": "http://schema.org/",
                  "@id": "https://www.rubensworks.net/#me", 
```

**Pull request merged**